src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +94,57 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page
+     *
+     * @param propertiesFiles
+     *            property files to read
+     */
+    public void parsePropertiesFiles(List<String> propertiesFiles) {
+
+        if (propertiesFiles.isEmpty()) {
+
+            return;
+
+        }
+
+        for (String propertyFile : propertiesFiles) {
```

Comment:
... if you have this but what about `null`?

Reply:
very good point this is fixed...by the way if a null pointer occurs anywhere in generateReporter...it fails two tests generateReports_GeneratesPages & generateReports_WithTrendsFile_GeneratesPages which is a good thing